generate-dockerfile-links-partial.sh

```diff
@@ -31,7 +31,7 @@ for repoDir in "${repoDirs[@]}"; do
 		# skip non-github.com for now
 		continue
 	fi
-	
+
```

Comment:
This one is just because we prefer to keep our whitespace properly indented for the enclosing block so it's easy to jump in and keep going.  I could be convinced to cave on this one (since it is often a source of contributor pain when they have to edit a script and their editor doesn't preserve these, for example), but the rest of this PR's future is definitely hazy. :wink: